WORKSPACE

```diff
@@ -254,3 +254,27 @@ http_archive(
     strip_prefix = "rules_testing-0.6.0",
     url = "https://github.com/bazelbuild/rules_testing/releases/download/v0.6.0/rules_testing-v0.6.0.tar.gz",
 )
+
+# For checking breaking changes to well-known types from the previous release version.
+load("//:protobuf_version.bzl", "PROTOBUF_PREVIOUS_RELEASE")
+
+http_archive(
+    name = "com_google_protobuf_previous_release",
+    strip_prefix = "protobuf-" + PROTOBUF_PREVIOUS_RELEASE,
+    url = "https://github.com/protocolbuffers/protobuf/releases/download/v{0}/protobuf-{0}.tar.gz".format(PROTOBUF_PREVIOUS_RELEASE),
+)
+
+http_archive(
+    name = "rules_buf",
```

Comment:
One of our bzlmod tests are failing, I think you need to add a dev_dependency to our MODULE file

Reply:
Sorry about that, I've updated `MODULE.bazel` and `WORKSPACE.bzlmod` such that the proto breaking tests should work with bzlmod enabled or disabled. I'm pretty new to bzlmod, let me know if I need to do anything else here.